domain-cc/cc-app/src/python_src/pydantic_models.py

```diff
@@ -16,14 +18,38 @@ def check_dc_for_cfi(cls, values):
         diagnostic_code = values.get("diagnostic_code")
 
         if claim_type == "claim_for_increase" and diagnostic_code is None:
-            raise ValueError(
-                "diagnostic_code is required for claim_type claim_for_increase"
+            raise HTTPException(
+                422, "diagnostic_code is required for claim_type claim_for_increase"
             )
         return values
 
 
+class Contention(BaseModel):
+    contention_text: str
+    diagnostic_code: Optional[int]  # only required for claim_type: "claim_for_increase"
+
+
+class VaGovClaim(BaseModel):
+    claim_id: int
+    form526_submission_id: int
+    claim_type: str  # "claim_for_increase" or "new"
```

Comment:
Should `claim_type` be an enum?

Reply:
I think for database models, makes a lot of sense to use an enum for fixed set of fields. 

For pydantic model which is indicative of inputs to the API in this case, do you think it makes sense for client to include
"claim_type": 1 on the request?

I feel that's less intuitive than the string indicating claim type from the client?

Reply:
Maybe a [StrEnum ](https://docs.python.org/3/library/enum.html#enum.StrEnum)would work well here?